### PR TITLE
Add params attribute to ExponentialSmoothing

### PR DESCRIPTION
**Pull request by norihitoishida**
Related to #192.

### Summary
Add `model.params` attribute to ExponentialSmoothing()

```
# Before
model = ExponentialSmoothing()
model.fit(train)
print(model.model.model.params) # {'smoothing_level': 0.5789473661331209, 'smoothing_slope': ...

# After
model = ExponentialSmoothing()
model.fit(train)
print(model.params) # {'smoothing_level': 0.5789473661331209, 'smoothing_slope': ...
```

### Other Information
Should I add any unittest for related to this PR?
I would like to get your opinions.
Thanks!

### CODE REVIEW

**Review by pennfranc — COMMENTED** [review]
Thanks for implementing this! I think it might be good to implement this attribute as a function `params()` with a `@property` attribute. That way we have a consistent flow of how the parameters are collected, (namely inside that function), and how they can be accessed (by calling that function as an attribute). We could also implement an abstract `params` function in the `ForecastingModel` superclass. What do you think?

**Comment by norihitoishida** [comment]
Thanks @pennfranc !
> I think it might be good to implement this attribute as a function `params()` with a `@property` attribute. 

Nice idea! I'll add that soon.

> We could also implement an abstract `params()` function in the `ForecastingModel` superclass. 

I agree with you.
I'd like to work on that once I finish implementing params in all models.

**Comment by norihitoishida** [comment]
Sorry, I couldn't know why the docs test fails...
Could someone please help me?

File: darts/models/exponential_smoothing.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ def __init__(self,
             `statsmodels.tsa.holtwinters.ExponentialSmoothing.fit()`.
             See `the documentation
             <https://www.statsmodels.org/stable/generated/statsmodels.tsa.holtwinters.ExponentialSmoothing.fit.html>`_.
+
+        Attributes
+        ----------
+        params
+            The parameters for the Exponential Smoothing model.
```
Comment (LeoTafti) [inline darts/models/exponential_smoothing.py:60]: > Sorry, I couldn't know why the docs test fails...
> Could someone please help me?

It seems like these lines are causing buildDocs to fail. Document directly in `params()` instead, e.g.:
```
    @property
    def params(self) -> Dict[Any, Any]:
        """
        The parameters for the Exponential Smoothing model.
        """
        return self._params
```

Though you might want to do that "one level higher" by defining and document a `params()` method in `ForecastingModel`, as @pennfranc suggested.

**Comment by norihitoishida** [comment]
Close this PR and I will made a single PR related to #192.

**Pull request closed without merging**